ddtrace/contrib/aiohttp/patch.py

```diff
@@ -48,13 +48,17 @@ def __init__(self, obj, pin):
 
     async def connect(self, req, *args, **kwargs):
         pin = Pin.get_from(self)
-        with pin.tracer.trace("%s.connect" % self.__class__.__name__):
+        with pin.tracer.trace("%s.connect" % self.__class__.__name__) as span:
+            # set component tag equal to name of integration
+            span.set_tag("component", config.aiohttp.integration_name)
```

Comment:
from ddtrace.internal.constants import COMPONENT

```suggestion
            span.set_tag(COMPONENT, config.aiohttp.integration_name)
```